### PR TITLE
Get the browser version from Selenoid status json

### PR DESCRIPTION
**Pull request by denysord88**

### CODE REVIEW

**Comment by denysord88** [comment]
@agozhiy @Atri0 please review this PR.

**Review by Atri0 — APPROVED** [review]
+1 LGTM

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java [hunk 1]
```diff
@@ -99,6 +98,19 @@ private static WebDriver getRemoteWebDriver(String driverVersion) {
     return remoteDriver;
   }
 
+  private static String getSelenoidBrowserVersion() {
+    JSONObject selenoidStatus = new JSONObject(given()
+        .get(PropertiesConst.DRILL_HOST + ":4444/status")
+        .getBody()
+        .print());
+    LinkedList<String> versions = new LinkedList<>();
+    versions.addAll(selenoidStatus.getJSONObject("browsers").getJSONObject(PropertiesConst.DRIVER_TYPE.toLowerCase()).keySet());
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: List has a suited constructor:
```suggestion
    LinkedList<String> versions = new LinkedList<>(
        selenoidStatus.getJSONObject("browsers")
        .getJSONObject(PropertiesConst.DRIVER_TYPE.toLowerCase())
        .keySet());
```

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java [hunk 2]
```diff
@@ -99,6 +98,19 @@ private static WebDriver getRemoteWebDriver(String driverVersion) {
...
+    if (versions.isEmpty()) {
+      throw new RuntimeException("Can't get Selenoid browser version! Status response was - " + selenoidStatus.toString());
+    }
+    return versions.get(versions.size() - 1);
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]:
```suggestion
    return versions.getLast();
```

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java [hunk 3]
```diff
@@ -148,6 +160,8 @@ public static void closeBrowser() {
     }
   }
 
+  private static LinkedList<String> parentWindows = new LinkedList<>();
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Move it back to the class fields please.

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java [hunk 4]
```diff
@@ -37,21 +38,19 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static io.restassured.RestAssured.given;
+
 public abstract class WebBrowser {
   public static String sessionId;
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Add a getter instead of doing a filed public.

**Review by agozhiy — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java (hunk 2 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java (hunk 1 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java (hunk 3 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java (hunk 4 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/WebBrowser.java]: Done

**Review by agozhiy — APPROVED** [review]

**Comment by agozhiy** [comment]
+1.

**Pull request merged**